firebase-firestore/src/main/java/com/google/firebase/firestore/DocumentSnapshot.java

```diff
@@ -313,6 +314,7 @@ public Object get(
    * @return The value at the given field or null.
    */
   @Nullable
+  @PublicApi
```

Comment:
Note that despite lacking this annotation, these methods were still showing up in our public api.

Reply:
`@PublicApi` was used when we obfuscated our codebase. I am wondering if Proguard does not obfuscate methods when overloads are marked `@PublicApi`. But we hopefully never have to find an answer to that question :)